File: src/domain/reviewInsight.domain.ts
export class ReviewInsightDomain {
  constructor(
    public id: string,
    public platform: string,
    public product_pk: string,
    public cautions: ReviewSummaryDomain[],
    public positives: ReviewSummaryDomain[]
  ) {}
}

export class ReviewSummaryDomain {
  constructor(public description: string, public reviewIds: string[]) {}
}

File: src/main/java/kr/okku/server/domain/ReviewInsightDomain.java
package kr.okku.server.domain;

import lombok.Builder;
import lombok.Data;

import java.util.List;

@Data
@Builder
public class ReviewInsightDomain {
    private String id;
    private String platform;
    private String productPk;
    private List<ReviewSummaryDomain> cautions;
    private List<ReviewSummaryDomain> positives;
    private String consSummary;
    private String prosSummary;
    private Integer reviewLen;
}
